presto-docs/src/main/sphinx/release/release-0.262.rst

```diff
@@ -0,0 +1,35 @@
+=============
+Release 0.262
+=============
+
+**Highlights**
+==============
+* Fix the issue of queries getting stuck in the queued state in disaggregated coordinator cluster.
+* Add support for simple TTL based scheduling based on worker availability estimates and query execution time estimates.
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix :func:`reduce_agg` to allow ``NULL`` as the result of input/combine functions, while also allowing only constant expressions and not ``NULL`` as the initial value.
+* Fix the issue of queries getting stuck in the queued state in disaggregated coordinator cluster.
+* Add default size limit (100MB) to build side of broadcast join.
+* Add support for simple TTL based scheduling where the scheduler picks workers for query execution based on their availability estimates and
+  query execution time estimates. This can be enabled by setting either the ``resource_aware_scheduling_strategy`` session property or the
+  ``experimental.resource-aware-scheduling-strategy`` configuration property to ``TTL``. See :pr:`16409`.
+* Add support to read adjusted queue size information from the ``v1/cluster`` endpoint.
+
+SPI Changes
+___________
+* Add support for custom Node TTL fetchers and Cluster TTL providers through the ``NodeTtlFetcher`` and ``ClusterTtlProvider`` interfaces respectively. See :pr:`16409`.
```

Comment:
lower case "node" and "cluster"? Also would be good to explain what they will be doing.